projects/ffmpeg/build.sh

```diff
@@ -0,0 +1,242 @@
+      export TARGET_DEM="-o /target_dem_cov target_dem_fuzzer.o main.o"
+fi
+
+# Disable UBSan vptr since several targets built with -fno-rtti.
+
+# Build dependencies.
+export FFMPEG_DEPS_PATH=/ffmpeg_deps
+rm -rf $FFMPEG_DEPS_PATH
+mkdir -p $FFMPEG_DEPS_PATH
+
+export PATH="$FFMPEG_DEPS_PATH/bin:$PATH"
+export LD_LIBRARY_PATH="$FFMPEG_DEPS_PATH/lib"
+
+cd /
+rm -rf alsa-lib-1.1.0/
+tar xf alsa-lib-1.1.0.tar
+cd alsa-lib-1.1.0
+./configure --prefix="$FFMPEG_DEPS_PATH" --enable-static --disable-shared
+make clean
+make -j$(nproc) all
+make install
+
+cd /fdk-aac
+autoreconf -fiv
+CXXFLAGS="$CXXFLAGS -fno-sanitize=shift-base,signed-integer-overflow" \
+./configure --prefix="$FFMPEG_DEPS_PATH" --disable-shared
+make clean
+make -j$(nproc) all
+make install
+
+cd /libva
+./autogen.sh
+./configure --prefix="$FFMPEG_DEPS_PATH" --enable-static --disable-shared
+make clean
+make -j$(nproc) all
+make install
+
+cd /libvdpau
+./autogen.sh
+./configure --prefix="$FFMPEG_DEPS_PATH" --enable-static --disable-shared
+make clean
+make -j$(nproc) all
+make install
+
+cd /libvpx
+LDFLAGS="$CXXFLAGS" ./configure --prefix="$FFMPEG_DEPS_PATH" \
+        --disable-examples --disable-unit-tests \
+        --size-limit=12288x12288 \
+        --extra-cflags="-DVPX_MAX_ALLOCABLE_MEMORY=1073741824"
+make clean
+make -j$(nproc) all
+make install
+
+cd /ogg
+./autogen.sh
+./configure --prefix="$FFMPEG_DEPS_PATH" --enable-static --disable-crc
+make clean
+make -j$(nproc)
+make install
+
+cd /opus
+./autogen.sh
+./configure --prefix="$FFMPEG_DEPS_PATH" --enable-static
+make clean
+make -j$(nproc) all
+make install
+
+cd /theora
+# theora requires ogg, need to pass its location to the "configure" script.
+CFLAGS="$CFLAGS -fPIC" LDFLAGS="-L$FFMPEG_DEPS_PATH/lib/" \
+      CPPFLAGS="$CXXFLAGS -I$FFMPEG_DEPS_PATH/include/" \
+      LD_LIBRARY_PATH="$FFMPEG_DEPS_PATH/lib/" \
+      ./autogen.sh
+./configure --with-ogg="$FFMPEG_DEPS_PATH" --prefix="$FFMPEG_DEPS_PATH" \
+      --enable-static --disable-examples
+make clean
+make -j$(nproc)
+make install
+
+cd /vorbis
+./autogen.sh
+./configure --prefix="$FFMPEG_DEPS_PATH" --enable-static
+make clean
+make -j$(nproc)
+make install
+
+cd /libxml2
+./autogen.sh --prefix="$FFMPEG_DEPS_PATH" --enable-static \
+      --without-debug --without-ftp --without-http \
+      --without-legacy --without-python
+make clean
+make -j$(nproc)
+make install
+
+# Remove shared libraries to avoid accidental linking against them.
+rm $FFMPEG_DEPS_PATH/lib/*.so
+rm $FFMPEG_DEPS_PATH/lib/*.so.*
+
+# Build ffmpeg.
+cd /ffmpeg
+
+PKG_CONFIG_PATH="$FFMPEG_DEPS_PATH/lib/pkgconfig" ./configure \
+        --cc=$CC --cxx=$CXX --ld="$CXX" \
+        --extra-cflags="-I$FFMPEG_DEPS_PATH/include $CFLAGS" \
+        --extra-ldflags="-L$FFMPEG_DEPS_PATH/lib $CFLAGS -std=c++11" \
+        --prefix="$FFMPEG_DEPS_PATH" \
+        --pkg-config-flags="--static" \
+        --libfuzzer=$LIB_FUZZING_ENGINE \
```

Comment:
We don't use `$LIB_FUZZING_ENGINE` variable

Reply:
removed